Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5544,152 +5544,158 @@ def get_security_profiles_command(security_profile: str = None):
         raise Exception('Please commit the instance prior to getting the security profiles.')
 
     human_readable = ''
-    content = []
     context = {}
     if 'spyware' in security_profiles and security_profiles['spyware'] is not None:
+        spyware_content = []
         profiles = security_profiles.get('spyware', {}).get('entry', {})
         if isinstance(profiles, list):
             for profile in profiles:
                 rules = profile.get('rules', {}).get('entry', [])
                 spyware_rules = prettify_profiles_rules(rules)
-                content.append({
+                spyware_content.append({
                     'Name': profile['@name'],
                     'Rules': spyware_rules
                 })
         else:
             rules = profiles.get('rules', {}).get('entry', [])
             spyware_rules = prettify_profiles_rules(rules)
-            content = [{
+            spyware_content = [{
                 'Name': profiles['@name'],
                 'Rules': spyware_rules
             }]
 
-        human_readable = tableToMarkdown('Anti Spyware Profiles', content)
-        context.update({"Panorama.Spyware(val.Name == obj.Name)": content})
+        human_readable = tableToMarkdown('Anti Spyware Profiles', spyware_content)
+        context.update({"Panorama.Spyware(val.Name == obj.Name)": spyware_content})
 
     if 'virus' in security_profiles and security_profiles['virus'] is not None:
+        virus_content = []
         profiles = security_profiles.get('virus', {}).get('entry', [])
         if isinstance(profiles, list):
             for profile in profiles:
                 rules = profile.get('decoder', {}).get('entry', [])
                 antivirus_rules = prettify_profiles_rules(rules)
-                content.append({
+                virus_content.append({
                     'Name': profile['@name'],
                     'Decoder': antivirus_rules
                 })
         else:
             rules = profiles.get('decoder', {}).get('entry', [])
             antivirus_rules = prettify_profiles_rules(rules)
-            content = [{
+            virus_content = [{
                 'Name': profiles['@name'],
                 'Rules': antivirus_rules
             }]
 
-        human_readable += tableToMarkdown('Antivirus Profiles', content)
-        context.update({"Panorama.Antivirus(val.Name == obj.Name)": content})
+        human_readable += tableToMarkdown('Antivirus Profiles', virus_content, headers=['Name', 'Decoder'])
```

Comment:
why `virus_content` have either `Rules` or 'Decoder'? why the table only have `Decoder`?

@bakatzir can you give your 2 cents here about the expected behavior?

Reply:
there can be application exceptions and theart id exceptions, but not always. so i would add them with a Null=True.
sent an image on slack.